robottelo/ui/products.py

```diff
@@ -57,18 +57,18 @@ def update(self, name, new_name=None, new_desc=None,
             if new_desc:
                 self.click(locators['prd.desc_edit'])
                 self.text_field_update(locators['prd.desc_update'], new_name)
-                self.click(common_locators['create'])
+                self.click(common_locators['save'])
```

Comment:
Just accidentaly found that bug while playing with adding a sync plan to package (tests are using adding package to syncplan however). As it's update func, of course there's no `create` button.